packages/extension/src/view/devtools/components/cookies/cookiesListing/cookieTableContainer/editableComponents/editableCheckBox.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies.
+ */
+import React, { useCallback, useState, useRef, useEffect } from 'react';
+
+/**
+ * Internal dependencies.
+ */
+import { useCookieStore } from '../../../../../stateProviders/syncCookieStore';
+import classNames from 'classnames';
```

Comment:
Should go under External dependencies, please check other places too.

Reply:
Fixed in latest commit.